puppeteer-scraper/INPUT_SCHEMA.json

```diff
@@ -125,6 +139,12 @@
             "description": "Scraper will download CSS stylesheets. Disabling this may speed up the scrape, but certain websites could stop working correctly.",
             "default": true
         },
+        "useLiveView": {
```

Comment:
Does it really make sense to have an option to disable the live view? The performance overheads of the live view is quite small - if the web page is not open, the live view doesn't consume too many resources. What do you think @mnmkng?

Reply:
I was just trying to maximize the efficiency of my compute units. Since I only use live view with testing, did not see the need for production scraping.